tests/spec/unit/lib/check_reqs.spec.js

```diff
@@ -46,25 +46,21 @@ describe('check_reqs', function () {
             );
         });
 
-        it('should throw error because version is not following semver-notated.', () => {
-            return checkTool('node', 'a.b.c').then(
-                () => fail('Expected promise to be rejected'),
-                err => expect(err).toEqual(new TypeError('Invalid Version: a.b.c'))
-            );
-        });
+        it('should throw error because version is not following semver-notated.', () => checkTool('node', 'a.b.c').then(
```

Comment:
That one wasn't caught by the rule I sent you. Here's a rule that requires tests to have block bodies:
```yaml
no-restricted-syntax: [2, 'CallExpression[callee.name=it] > .arguments:nth-child(2) > .body:not(BlockStatement)']
```